--- origsrc/lame-398/configure.in	2008-06-29 16:40:06.000000000 -0500
+++ src/lame-398/configure.in	2008-07-11 18:24:06.078125000 -0500
@@ -706,6 +706,7 @@
 
 
 
+AC_PATH_PROG(ECHO, echo, no)
 AC_PATH_PROG(NASM, nasm, no)
 case "${NASM}" in
 no)
--- origsrc/lame-398/frontend/mp3rtp.c	2005-11-29 15:21:01.000000000 -0600
+++ src/lame-398/frontend/mp3rtp.c	2008-07-11 18:47:04.296875000 -0500
@@ -12,8 +12,6 @@
 
 #ifdef HAVE_CONFIG_H
 # include <config.h>
-/* std int types already defined in config.h */
-#define _STDINT_H
 #endif
 
 #ifdef STDC_HEADERS
--- origsrc/lame-398/frontend/rtp.c	2005-11-22 16:15:39.000000000 -0600
+++ src/lame-398/frontend/rtp.c	2008-07-11 18:47:11.500000000 -0500
@@ -2,8 +2,6 @@
 
 #ifdef HAVE_CONFIG_H
 # include <config.h>
-/* std int types already defined in config.h */
-# define _STDINT_H
 #endif
 
 #ifndef __GNUC__
